save-demo/src/main/kotlin/com/saveourtool/save/demo/controller/DemoController.kt

```diff
@@ -25,20 +29,41 @@ import kotlin.io.path.createTempDirectory
     Tag(name = "demo"),
 )
 @RestController
-@RequestMapping("/demo/api/{organizationName}/{projectName}")
+@RequestMapping("/demo/api")
 class DemoController(
     private val demoService: DemoService,
     private val demoCliRunnerFactory: CliRunnerFactory,
 ) {
     private val tmpDir = createTempDirectory("demo-")
 
+    /**
+     * @return all [DemoDto]s as [Flux]
+     */
+    @GetMapping("/all")
+    fun all(): Flux<DemoDto> = blockingToFlux {
+        demoService.getAllDemos().map { it.toDto() }
+    }
+
+    /**
+     * @param organizationName
+     * @param projectName
+     * @return configName or empty response
+     */
+    @GetMapping("/{organizationName}/{projectName}/config-name")
+    fun configName(
```

Comment:
Can be replaced with `ManagementControllers`'s getInfo()

Reply:
will make it as public to reuse it for `config-name`

Reply:
actually, will leave it for you. will use endpoint from backend `/api/v1/demo`?

Reply:
Well, let's leave as is. Not that important